opentelemetry-collector/chart/values.yaml

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+installCRDs: true
+
+opentelemetry-operator:
+  admissionWebhooks:
+    failurePolicy: 'Ignore'
+  manager:
+    image:
+      repository: otel/opentelemetry-collector
+      tag: 0.98.0
+    collectorImage:
+      repository: otel/opentelemetry-collector-contrib
+      tag: 0.99.0
```

Comment:
Can be omitted. The defaults are already here: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/aafd6f046a5dca9c0ff07cad55de8d2e1cb317c8/charts/opentelemetry-operator/values.yaml#L40-L45
```suggestion
```